velox/docs/functions/spark/decimal.rst

```diff
@@ -114,6 +114,9 @@ Returns NULL when the actual result cannot be represented with the calculated de
 
 Decimal Functions
 -----------------
+.. spark:function:: floor(x: decimal(p, s)) -> r: decimal(pr, 0) 
+
+    Returns ``x`` rounded down to the type ``decimal(min(p - s + 1, 38), 0)``.
```

Comment:
I assume when the 's' is 0, the output precision is 'p' in Spark. The formula used by Presto decimal floor is correct.

> "min(38, {p} - {s} + min({s}, 1))"

Reply:
Yes, you are correct, I missed the case 0.